### PR TITLE
All JDI homework files are ready for review

### PR DESCRIPTION
**Pull request by evgenii-liashenko**
I have completed the seventh homework (JDI) and I am looking forward to your feedback

### CODE REVIEW

File: src/test/java/ru/training/at/hw7/entities/User.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    private String password;
+    private String fullName;
+
+    //Constructors
+    public User() {
+        System.out.println("Creating default User object");
+        Properties credentials = TestDataLoader.getCredentials();
+        this.name = credentials.getProperty("username");
+        this.password = credentials.getProperty("password");
+        this.fullName = "Roman Iovlev";
```
Comment (shoggy) [inline src/test/java/ru/training/at/hw7/entities/User.java]: I think `fullName` is a parameter that should be stored in same way as `name/password`

File: src/test/java/ru/training/at/hw7/testcases/MetalsAndColorsPageTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    public void loadMetalsAndColorsPage() {
+        JdiWebsite.homePage.menuBar.metalsAndColors.click();
+        JdiWebsite.metalsAndColorsPage.checkOpened();
+    }
+
+    //"3. Fill form Metals & Colors by data"
+    // "Submit form Metals & Colors"
+    @Test(dependsOnMethods = {"loadMetalsAndColorsPage"}, dataProvider = "metalsAndColorsPageTestData", dataProviderClass = TestDataLoader.class)
+    public void enterTestData(TestDataSet testDataSet) {
+        //Setting up the form
```
Comment (shoggy) [inline src/test/java/ru/training/at/hw7/testcases/MetalsAndColorsPageTest.java:40]: Why multiple tests?

File: src/test/java/ru/training/at/hw7/testdata/TestDataLoader.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        catch (FileNotFoundException e){
+            System.out.println("Failed to load file from path " + filePath);
+            e.printStackTrace();
+        }
+        return extractedData;
+    }
+
+    //DataProvider method
+    @DataProvider(name = "metalsAndColorsPageTestData")
+    public static Object[][] dpMethod() {
```
Comment (shoggy) [inline src/test/java/ru/training/at/hw7/testdata/TestDataLoader.java]: method name should be more descriptive

File: src/test/java/ru/training/at/hw7/configs/testng/homework7.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
```
Comment (shoggy) [inline src/test/java/ru/training/at/hw7/configs/testng/homework7.xml]: I think `resources` is more appropriate place for this file

**Review by shoggy — CHANGES_REQUESTED** [review]

File: src/test/java/ru/training/at/hw7/entities/User.java (hunk 1 above)
Comment (evgenii-liashenko) [reply src/test/java/ru/training/at/hw7/entities/User.java]: Fixed. Now it is in userdata.properties file

File: src/test/java/ru/training/at/hw7/testcases/MetalsAndColorsPageTest.java (hunk 2 above)
Comment (evgenii-liashenko) [reply src/test/java/ru/training/at/hw7/testcases/MetalsAndColorsPageTest.java:40]: We have discussed that in Teams

File: src/test/java/ru/training/at/hw7/testdata/TestDataLoader.java (hunk 3 above)
Comment (evgenii-liashenko) [reply src/test/java/ru/training/at/hw7/testdata/TestDataLoader.java]: I agree. This method has been renamed to jsonTestDataProvider(). New name suggests where test values come from

File: src/test/java/ru/training/at/hw7/configs/testng/homework7.xml (hunk 4 above)
Comment (evgenii-liashenko) [reply src/test/java/ru/training/at/hw7/configs/testng/homework7.xml]: I have moved it to src/test/resources/testng.configs/homework7.xml

**Review by shoggy — APPROVED** [review]

**Pull request merged**